oidc-authenticator/src/main/java/org/xwiki/contrib/oidc/auth/internal/OIDCUserManager.java

```diff
@@ -234,7 +237,21 @@ private int sendLogout(Endpoint logoutEndpoint, IDTokenClaimsSet idToken) throws
 
     private void checkAllowedGroups(UserInfo userInfo) throws OIDCException
     {
-        List<String> providerGroups = getClaim(this.configuration.getGroupClaim(), userInfo);
+    	List<String> providerGroups = null;
+        Object providerGroupsObj = getClaim(this.configuration.getGroupClaim(), userInfo);
+        if (this.configuration.isGroupCommaDelimited()) {        
+        	providerGroups = Arrays.asList(StringUtils.split(providerGroupsObj.toString(), ","));
+        } else {
+        	providerGroups = (List<String>)providerGroupsObj;
+        }
+        String groupPrefix = this.configuration.getGroupPrefix();
+        if (!StringUtils.isEmpty(groupPrefix)) {
+        	providerGroups = providerGroups.stream()
```

Comment:
This project's codestyle is to use white spaces and not tabs for indentation.

Reply:
Sorry, I missed setting up the IDE. I'll fix this.